pkgs/tools/nix/nixos-render-docs/default.nix

```diff
@@ -1,15 +1,11 @@
 { lib
 , stdenv
 , python3
-, python3Minimal
 , runCommand
 }:
 
 let
-  # python3Minimal can't be overridden with packages on Darwin, due to a missing framework.
-  # Instead of modifying stdenv, we take the easy way out, since most people on Darwin will
-  # just be hacking on the Nixpkgs manual (which also uses make-options-doc).
-  python = ((if stdenv.isDarwin then python3 else python3Minimal).override {
+  python = (python3.override {
```

Comment:
In nixos/tests/installer.nix you removed the `.override { self=python; includeSiteCustomize = true; }` part, but not here. Any reason why?

Reply:
Good catch, this is no longer needed.